azure-pipelines.yml

```
@@ -181,22 +184,27 @@ jobs:
     vmImage: 'macOS-10.15'
   strategy:
     matrix:
-      Python36:
-        python.version: '3.6'
-        python.org.version: '3.6.8'
-      Python37:
-        python.version: '3.7'
-        python.org.version: '3.7.7'
-      Python38:
-        python.version: '3.8'
-        python.org.version: '3.8.2'
-      Python39:
-        python.version: '3.9'
-        python.org.version: '3.9.0'
+      ${{ if eq(variables.buildWheel, True) }}:
+        Python36:
+          python.version: '3.6'
+          python.org.version: '3.6.8'
+        Python37:
+          python.version: '3.7'
+          python.org.version: '3.7.7'
+        Python38:
+          python.version: '3.8'
+          python.org.version: '3.8.2'
+        Python39:
+          python.version: '3.9'
+          python.org.version: '3.9.0'
+      ${{ if ne(variables.buildWheel, True) }}:
```

```suggestion
      ${{ if eq(variables.buildWheel, False) }}:
```
maybe just a matter of taste.. but this seems a bit more readable